main.tf

```diff
@@ -9,6 +9,12 @@ resource "aws_instance" "webserver" {
   tags = {
     Name = "webserver"
   }
+  ebs_optimized = true
+
+  metadata_options {
+    http_endpoint = "disabled"
+    http_tokens   = "required"
+  }
 }
 
 resource "aws_s3_bucket" "accuricsbucketdemo" {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure access logging is enabled for AWS S3 Buckets**
 
**Vulnerability**
AWS S3 Buckets have access logging disabled. Access log information can be useful in security and access audits.
 
**Remediation**
By default, Amazon S3 doesn't collect server access logs. When you enable logging, Amazon S3 delivers access logs for a source bucket to a target bucket that you choose. The target bucket must be in the same AWS Region as the source bucket and must not have a default retention period configuration.

source: accurics-localhost_harshit